src/main/pages/che-7/contributor-guide/proc_adding-support-for-a-new-language.adoc

```diff
@@ -1,27 +0,0 @@
-[id="adding-support-for-a-new-language_{context}"]
```

Comment:
The file shouldn't be removed -- that would defeat the purpose; we want the topic published.